generators/delimited-schema/main.py

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env python3
+import sys
+import os
+import csv
+import time
+from unicode_csv import UnicodeCsvReader
+
+def main():
+    usage = """Origins Delimited Csv Schema Generator.
+
+    Usage:
+        delimited-schema <file_name> [--encoding=<encoding>] [--delimiter=<delimiter>] [--operation=<assert>|<retract>] [--valid_time=<valid_time>]
+
+    Options:
+        -h --help  Show usage info.
+        --encoding=<encoding>  The text encoding type of the file being read.
+        --delimiter=<delimiter>  The character used to separate items in the csv file.
+        --operation=<assert>|<retract>  The origins operation intended for the new facts. (assert|retract)
+        --valid_time=<valid_time>  The valid time for new facts.
```

Comment:
The operation and valid_time options can be removed since the `origins generate` command supports those options already.